src/main/java/com/endava/internship/warmup/domain/service/ArrayProcessorWithForLoops.java

```diff
@@ -46,7 +69,12 @@ public boolean allMatch(final String[] input,
      */
     @Override
     public int[] copyValues(int[] input, int startInclusive, int endExclusive) throws IllegalArgumentException {
```

Comment:
Recommend here to use the try-catch blocks (throwing the Exception in the catch block)